### DISCUSSION TITLE
Add <meta property="og:locale:alternate" content="" /> Support to Metadata Object and generateMetadata Function

### DISCUSSION THREAD

**Discussion by MicaelPiresCom** [vercel/next.js#58903] [Ideas]
### Goals

1. Add `<meta property="og:locale:alternate" content="" />` support to `metadata` object and `generateMetadata` function.

### Background

It's not possible to add it via `other: { custom: 'meta' }`, as it returns `<meta name="custom" ...` instead of `<meta property="custom" ...` as it is needed in Open Graph tags.

### Proposal

The `openGraph: { locale: '' }` can be changed to support either a string or an object. The object could support `locale` and `alternates` which would be an array of strings.

BUT

If there is already `alternates (<link rel="alternate" ...)` defined, just reuse that data and add the og:locale:alternate tags automatically.

**Comment by invisal** [comment]
I have been struggling with this as well. Since we need to implement https://developers.facebook.com/docs/marketing-api/catalog/guides/microdata-tags/ this and it is at the moment not possible with `generateMetadata`

**Comment by Shark43** [comment]
Anyone have an update on this? i'm also keep struggling with this
